lib/data_hygiene/attachment_attribute_updater.rb

```diff
@@ -0,0 +1,40 @@
+module DataHygiene
+  class AttachmentAttributeUpdater
+    def initialize(attachment, dry_run:)
+      @attachment = attachment
+      @dry_run = dry_run
+    end
+
+    def call
+      valid_number = fix_command_paper_number(attachment.command_paper_number)
+
+      unless AttachmentValidator.new.command_paper_number_valid?(valid_number)
+        raise DataHygiene::AttachmentAttributeNotFixable.new
+      end
+
+      return valid_number if dry_run
+
+      attachment.update(command_paper_number: valid_number)
```

Comment:
I think we should use `update!` here, just in case there's a problem with the save operation.

Reply:
If something does go wrong, we'll want this to be in a transaction (that'll have to go in the rake task), in case we lose sight of the ones we've changed.